app/src/main/res/layout/fragment_headline.xml

```diff
@@ -16,6 +16,16 @@
 
         <FrameLayout
             android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+        </android.support.v4.widget.SwipeRefreshLayout>
+
+        <ProgressBar
+            android:id="@+id/progressBar"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:layout_gravity="center"
+            android:visibility="visible" />
+    </FrameLayout>
```

Comment:
what's this? How can frame layout  be closed here. You can clearly see that the below lines aren't present inside any of the tags